README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://github.com/mobie/mobie-viewer-fiji/actions/workflows/build.yml/badge.svg)](https://github.com/mobie/mobie-viewer-fiji/actions/workflows/build.yml)
+
 [![](https://github.com/mobie/mobie-viewer-fiji/actions/workflows/build-main.yml/badge.svg)](https://github.com/mobie/mobie-viewer-fiji/actions/workflows/build-main.yml)
```

Comment:
I forgot to remove old badges. This one here was already invalid before this PR, so let's remove it.

```suggestion
```

Reply:
Thanks!